src/aics-image-viewer/components/App/index.tsx

```diff
@@ -584,7 +584,7 @@ const App: React.FC<AppProps> = (props) => {
     view3d.setCameraMode(viewerSettings.viewMode);
   }, []);
 
-  // Hook to trigger image load: on mount, when image source props/state change (`cellId`, `imageType`, `time`)
+  // Hook to trigger image load: on mount, when image source props/state change (`cellId`, `imageType`, data)
```

Comment:
add `` around "data"?

Reply:
This was meant to capture both `rawData` and `rawDims` - changes to either indicate the presence of new *data* needs to be loaded into the viewer (though they'd likely both change at the same time). But you're right that it's confusing - updated.